diff --git a/src/libssh2_setup.h b/src/libssh2_setup.h
index 0f17a18..165ad28 100644
--- a/src/libssh2_setup.h
+++ b/src/libssh2_setup.h
@@ -14,6 +14,10 @@
 #include <_mingw.h>
 #endif
 
+#ifdef __MVS__
+#include <sys/time.h>
+#endif
+
 /* Configuration provided by build tools (autotools and CMake),
    and via platform-specific directories for os400 and vms */
 #if defined(HAVE_CONFIG_H) || defined(__OS400__) || defined(__VMS)
